src/Sylius/Bundle/ApiBundle/spec/Applicator/PaymentStateMachineTransitionApplicatorSpec.php

```diff
@@ -52,4 +52,31 @@ function it_throws_exception_if_cannot_complete_payment(
             ->during('complete', [$payment])
         ;
     }
+
+    function it_refunds_payment(
+        StateMachineInterface $stateMachine,
+        PaymentInterface $payment,
+    ): void {
+        $this->beConstructedWith($stateMachine);
```

Comment:
```suggestion
```
it should not be needed, just use the same argument name here and in `let` method